Client/Frontend/Browser/Onboarding/OnboardingNavigationController.swift

```diff
@@ -55,7 +56,20 @@ class OnboardingNavigationController: UINavigationController {
                     if progress == .rewards || progress == .ads {
                         return BraveAds.isCurrentLocaleSupported() ? [.adsCountdown] : [.rewardsAgreement]
                     }
-                    return BraveAds.isCurrentLocaleSupported() ? [.searchEnginePicker, .shieldsInfo, .rewardsAgreement, .adsCountdown] : [.searchEnginePicker, .shieldsInfo, .rewardsAgreement]
+                    
+                    var newUserScreens: [Screens] = [.searchEnginePicker, .shieldsInfo, .rewardsAgreement]
+                    
+                    if BraveAds.isCurrentLocaleSupported() {
+                        newUserScreens.append(.adsCountdown)
+                    }
+                    
+                    // FIXME: Once iOS14 launches replace `hasStrings` with `UIPasteboard.DetectionPattern` api.
```

Comment:
If we don't already have an issue made for fixing this and the other comment I made in iOS 14 then we might want to do that :)

Reply:
I will make a ticket, good idea